napari/utils/transforms/transforms.py

```diff
@@ -55,9 +55,9 @@ def inverse(self) -> 'Transform':
                 trans._('Inverse function was not provided.', deferred=True)
             )
 
-    def compose(self, transform: 'Transform') -> 'Transform':
+    def compose(self, transform: Optional['Transform']) -> 'Transform':
         """Return the composite of this transform and the provided one."""
-        return TransformChain([self, transform])
+        return self if transform is None else TransformChain([self, transform])
```

Comment:
My suggestion is to revert the change here

```suggestion
        return TransformChain([self, transform])
```

and special case them in the interaction box code: https://github.com/napari/napari/pull/4868/files#r930376668